CODEOWNERS

```diff
@@ -135,14 +135,14 @@ Kconfig*                                  @tejlmand
 /modules/                                 @tejlmand
 /modules/hostap/                          @krish2718 @jukkar @rado17 @sachinthegreen @rlubos
 /modules/mcuboot/                         @de-nordic @nordicjm
-/modules/cjson/                           @simensrostad @plskeggs @sigvartmh
+/modules/cjson/                           @nrfconnect/ncs-cia @plskeggs @sigvartmh
 /modules/tfm/                             @frkv @Vge0rge @vili-nordic @joerchan @SebastianBoe @mswarowsky
 /modules/trusted-firmware-m/              @frkv @Vge0rge @vili-nordic @joerchan @SebastianBoe @mswarowsky
 /samples/                                 @nrfconnect/ncs-test-leads
-/samples/net/mqtt/                        @simensrostad
-/samples/net/udp/                         @simensrostad
-/samples/sensor/bh1749/                   @gregersrygg
-/samples/sensor/bme68x_iaq/               @maxd-nordic
+/samples/net/mqtt/                        @nrfconnect/ncs-cia
```

Comment:
We own all the samples under NET. Could you create an entry for that while you are at it?